### PR TITLE
SEOImages: User Story 1463880, Part 13

### PR DESCRIPTION
**Pull request by damabe**
## Summary

Updating image file names and references to improve SEO.

Files #1463880

Contributes to #10839

### CODE REVIEW

**Comment by damabe** [comment]
@mairaw This PR is ready for review. CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019031600009/Report/

File: docs/core/porting/project-structure.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ The following describes several ways to add support for .NET Core for this repos
 
 Reorganize the repository so that any existing *\*.csproj* files are removed and a single *\*.csproj* file is created that targets multiple frameworks. This is a great option because a single project is able to compile for different frameworks. It also has the power to handle different compilation options and dependencies per targeted framework.
 
-![Create an csproj that targets multiple frameworks](media/project-structure/project.csproj.png)
+![Create an csproj that targets multiple frameworks](./media/project-structure/project-csproj-frameworks.png)
```
Comment (mairaw) [inline docs/core/porting/project-structure.md]: Image name suggestion:
```suggestion
![Create an csproj that targets multiple frameworks](./media/project-structure/multi-targeted-project.png)
```

File: docs/core/porting/project-structure.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ Changes to note are:
 
 If there are existing projects that target older frameworks, you may want to leave these projects untouched and use a .NET Core project to target future frameworks.
 
-![.NET Core project with existing project in different folder](media/project-structure/project.csproj.different.png)
+![.NET Core project with existing project in different folder](./media/project-structure/project-csproj-different.png)
```
Comment (mairaw) [inline docs/core/porting/project-structure.md]:
```suggestion
![.NET Core project with an existing project in a different folder](./media/project-structure/separate-projects-same-source.png)
```

File: docs/framework/wpf/controls/gridview-overview.md [hunk 3]
```diff
@@ -32,10 +30,8 @@ ms.assetid: b2d02267-32b3-40ce-8e9f-06972d8749d9
  [!code-xaml[ListViewCode#ListViewEmployee](~/samples/snippets/csharp/VS_Snippets_Wpf/ListViewCode/CSharp/Window1.xaml#listviewemployee)]  
   
  The following illustration shows the table that the previous example creates.
```
Comment (mairaw) [inline docs/framework/wpf/controls/gridview-overview.md]:
```suggestion
 The following illustration shows the table that the previous example creates. The GridView control displays data from an ItemsSource object:
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Looks great @damabe. A few suggestions...

**Comment by damabe** [comment]
@mairaw Thanks! Changes incorporated. Updated CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019031700052/Report/

**Review by mairaw — APPROVED** [review]

**Pull request merged**